_reports/saksham.md

```diff
@@ -0,0 +1,266 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/SakshamBansal
```

Comment:
see updates to the example.  needs date, categories, and not layout or permalink, etc.